sqeduler.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "pry", "~> 0"
   gem.add_development_dependency "rake", "~> 10"
   gem.add_development_dependency "rspec", "~> 3.3"
-  gem.add_development_dependency "rubocop", "~> 0.24"
```

Comment:
So it was already pinned to 0.24... so how were the builds failing? That doesn't really make sense. Do you have a link to a failed build?